### ISSUE TITLE
ssl_generate.yml 

### ISSUE THREAD

**Issue by sqanto** [cmacrae/ansible-sensu#42]
Maybe its the same problem : ssl download error #41

fatal: [xxxx.xxxx.xxxx]: FAILED! => {"changed": false, "failed": true, "msg": "Destination /etc/sensu/ssl_generation not writable"}

**Comment by sqanto** [comment]
maybe its help ? 

```
- name: Untar the ssl_certs tarball from sensuapp.org
  unarchive:
  args:
     src: "{{ sensu_config_path }}/ssl_generation/sensu_ssl_tool.tar"
     dest: "{{ sensu_config_path }}/ssl_generation/"
     copy: no  <---- Option
     creates: "{{ sensu_config_path }}/ssl_generation/sensu_ssl_tool"
```

**Comment by smuth4** [comment]
I believe this is due to https://github.com/ansible/ansible/issues/16952. Prefixing all tasks within the block with `when: sensu_master|bool` seems to work around it.

**Comment by smuth4** [comment]
This has been fixed in the latest version of ansible (2.1.2.0), closing.
